beacon_node/beacon_chain/src/test_utils.rs

```diff
@@ -1030,7 +1032,7 @@ where
             *state.get_block_root(target_slot)?
         };
 
-        Ok(Attestation {
+        Ok(Attestation::Base(AttestationBase {
             aggregation_bits: BitList::with_capacity(committee_len)?,
```

Comment:
maybe add `TODO(electra): make test fork-agnostic` if it makes sense

Reply:
af73dde